adal/src/androidTest/java/com/microsoft/aad/adal/OauthTests.java

```diff
@@ -73,7 +77,7 @@ public void testParseIdTokenPositive() throws UnsupportedEncodingException, Auth
         assertEquals("emailField", actual.getEmail());
         assertEquals("idpProvider", actual.getIdentityProvider());
         assertEquals("53c6acf2-2742-4538-918d-e78257ec8516", actual.getObjectId());
-        assertTrue(1387227772 == actual.getPasswordExpiration());
+        assertTrue(Util.TEST_PASSWORD_EXPIRATION == actual.getPasswordExpiration());
```

Comment:
can we flip the l-value and the r-value here, everywhere else in this file constants are on the right side.